src/helper/selection-tools/bounding-box-tool.js

```diff
@@ -23,6 +25,7 @@ const BoundingBoxModes = keyMirror({
     ROTATE: null,
     MOVE: null
 });
+let anchorIcon;
```

Comment:
This global variable seems like a strange pattern. Why isn't it added as a property of the BoundingBoxTool prototype the same way things like `this.boundsPath` are?

Reply:
That was so that we would only need to load anchorIcon from SVG once! I fixed the code where it loads anchorIcon so it actually does that.